lib/sandthorn_sequel_projection.rb

```diff
@@ -22,7 +21,7 @@ class << self
     require 'delegate'
     extend Forwardable
 
-    def_delegators :configuration, :batch_size, :event_stores
+    def_delegators :configuration, :batch_size, :event_store
```

Comment:
Now it goes via Sandthorn to get the events.

```
module SandthornSequelProjection
  class EventStore
    def initialize(name = :default)
      @name = name || :default
    end

    def get_events(*args)
      keywords = args.pop || {}
      keywords[:event_store] = @name
      Sandthorn.get_events(keywords)
    end
  end
end
```

I would like to get the events from the driver directly.